### DISCUSSION TITLE
Any update on helix-editor on crates.io?

### DISCUSSION THREAD

**Discussion by jkemp814** [helix-editor/helix#7038] [General] [answered]
There have been placeholders on crates.io for quite some time. Does anyone know when 'helix' will be available via crates.io?
Would certainly simplify updating. I wonder why it was not put up there before becoming a flatpak?

cargo search helix
helix-core = "0.0.0"      # This package is a placeholder for the future helix-editor crate.
helix-editor = "0.0.0"    # This package is a placeholder for the future helix-editor crate.
helix-lsp = "0.0.0"       # This package is a placeholder for the future helix-editor crate.
helix-syntax = "0.0.0"    # This package is a placeholder for the future helix-editor crate.
helix-term = "0.0.0"      # This package is a placeholder for the future helix-editor crate.
helix-tui = "0.0.0"       # This package is a placeholder for the future helix-editor crate.
helix-view = "0.0.0"      # This package is a placeholder for the future helix-editor crate.

**Comment by the-mikedavis** [comment, marked as answer]
Publishing to crates is not planned, see https://github.com/helix-editor/helix/issues/42

**Comment by jkemp814** [comment]
How do we go about removing those placeholders on crates.io? Should I forward this response to the maintainer over on crates.io, so they will know the intent is to not package it as a crate? Seems to be wasting a small amount of their resources by listing it and reserving those names. Gives the impression it will be done, when it's not.

**Reply by pascalkuthe** [reply]
Nothing is ever removed from crates.io if it's not malware and placeholders like this are perfectly valid to prevent supply chain attacks

**Reply by gabydd** [reply]
The maintainer on crates.io (blaz/archseer is also the creator of helix) so this intent not to package is there's

**Reply by archseer** [reply]
Yes, I pre-emptively took the crate names since it's a first come first serve basis. Someone could have uploaded a malicious package if we didn't squat the name.
